.travis.yml

```diff
@@ -17,6 +17,10 @@ env:
     - LANG=en_US.UTF-8
     - LC_ALL=en_US.UTF-8
     - secure: 2c0250fb-dfb1-45c9-abaf-cd3d874326c5
+    - secure: 2c0250fb-dfb1-45c9-abaf-cd3d874326c5
```

Comment:
Should these lines be repeated? I'm not sure how the secured version of travis works, but this doesn't seem right...